packages/cli/src/Init.ts

```diff
@@ -211,9 +211,9 @@ export class Init implements Command {
       url = args['--url']
     } else if (args['--datasource-provider']) {
       const providerLowercase = args['--datasource-provider'].toLowerCase()
-      if (!['postgresql', 'mysql', 'sqlserver', 'sqlite', 'mongodb'].includes(providerLowercase)) {
+      if (!['postgresql', 'mysql', 'sqlserver', 'sqlite', 'mongodb', 'cockroachdb'].includes(providerLowercase)) {
         throw new Error(
-          `Provider "${args['--datasource-provider']}" is invalid or not supported. Try again with "postgresql", "mysql", "sqlite", "sqlserver" or "mongodb".`,
+          `Provider "${args['--datasource-provider']}" is invalid or not supported. Try again with "postgresql", "mysql", "sqlite", "sqlserver", "mongodb", or "cockroachdb".`,
```

Comment:
It may be worth creating a small utility that transforms the array of supported datasource providers to a stringified version to append after the "Try again with " string.

Reply:
I think that would be an ok optimization to do along the way, on the other hand another stacking PR (towards this one, that will be change to `main` automatically when this here is merged) might be clearer.